### ISSUE TITLE
need to sum particular column of grid

### ISSUE THREAD

**Issue by dahn-d** [aardappel/treesheets#37]
hi, i ve fallen in love with treesheets. u r awesome. i d like to collaborate after uni semester end

i want to ask a question
i have a Nx2 grid of tasks i wanna do today and aproximate time in hours i need to spend
![ss](https://cloud.githubusercontent.com/assets/6011365/10664328/37ee9546-78cb-11e5-8348-c5358009652c.png)  
and its useful to know the total to get all things done today
but i get summation of both cols
and i dont want to split tasks and times into separate subcells
**a**. do u see some workaround?
**b**. if not **a** then is it useful thing to implement? see the example:
![ss](https://cloud.githubusercontent.com/assets/6011365/10664586/670eef54-78cd-11e5-8ce4-dac1dd034b8d.png)
... like some simple indexing syntax or smth. i know treesheets is not Excel but i think my idea isn't an overengineering

what u think?

**Comment by aardappel** [comment]
The scripting language is definitely very incomplete. Selecting a row or column is definitely something that should be added, but it should be separate from sum, e.g. `2D_grid | column | 1 | sum` or something like that.

**Comment by dahn-d** [comment]
hmm... im leaving here this screenshot just for an idea. is it rational?
![ss](https://cloud.githubusercontent.com/assets/6011365/10822155/a2fd878c-7e5e-11e5-879f-4ea4d50ce711.png)

**Comment by aardappel** [comment]
That wouldn't work with the current language, no, since the arguments to sum must all sit inside a single cell. A similar function can be implemented of course.. like sum-all or something.

**Comment by dahn-d** [comment]
i see now why. because Evaluator->Execute are designed for one operation and 0,1,2 or 3 context cells. so without redesigning the structure, is it adequate to create a wrapper cell containing all the cells to left (or top) and feed it to Execute? i've skimmed through code, but was not able to see how to do it

**Comment by aardappel** [comment]
Yup, that would be possible, but it would be different from how everything else works.
